katana/wapps/projects/suite_utils/defaults.py

```diff
@@ -0,0 +1,132 @@
+from utils.dict_utils import invert_dict
```

Comment:
The directory Naming convention.  katana/wapps/project/project_utils/
rename suite_utils to project_utils